charter.html

```diff
@@ -0,0 +1,290 @@
+      <li>Extensions to Canvas/WebGPU/WebGL APIs for HDR support</li>
+    </ul>
+    <h3 id="non-normative-reports">
+      Non-Normative Reports
+    </h3>
+    <ul>
+      <li>CG report</li>
+      <li>Requirements for adding HDR support in PNG</li>
+      <li>Explainer for adding floating point support to HTML canvas</li>
+    </ul>
+    <h3 id="test-suites">
+      Test Suites and Other Software
+    </h3>
+    <p>
+      The group MAY produce test suites to support the Specifications. Please
+      see the GitHub LICENSE file for test suite contribution licensing
+      information.
+    </p>
+    <h2 id="liaisons">
+      Dependencies or Liaisons
+    </h2>
+    <p>
+      The group expects to liaise with the following groups:
+      <ul>
+        <li><a href="https://www.w3.org/groups/wg/htmlwg">W3C HTML WG</a></li>
+        <li><a href="https://www.w3.org/groups/wg/png">W3C PNG WG</a></li>
+        <li><a href="https://www.w3.org/2011/webtv/wiki/Main_Page">W3C Media and Entertainment Interest Group</a></li>
+        <li><a href="https://www.w3.org/groups/wg/gpu">W3C GPU for the Web Working Group</a></li>
+        <li><a href="https://www.w3.org/media-wg/">W3C Media Working Group</a></li>
+        <li><a href="https://github.com/whatwg/html">WHATWG HTML Workstream</a></li>
+        <li><a href="https://github.com/AOMediaCodec">Alliance for Open Media</a></li>
+        <li><a href="https://github.com/KhronosGroup/WebGL">Khronos WebGL</a></li>
+      </ul>
+    </p>
+    <h2 id="process">
+      Community and Business Group Process
+    </h2>
+    <p>
+      The group operates under the <a href=
+      "https://www.w3.org/community/about/agreements/">Community and Business
+      Group Process</a>. Terms in this Charter that conflict with those of the
+      Community and Business Group Process are void.
+    </p>
+    <p>
+      As with other Community Groups, W3C seeks organizational licensing
+      commitments under the <a href=
+      'http://www.w3.org/community/about/agreements/cla/'>W3C Community
+      Contributor License Agreement (CLA)</a>. When people request to
+      participate without representing their organization's legal interests,
+      W3C will in general approve those requests for this group with the
+      following understanding: W3C will seek and expect an organizational
+      commitment under the CLA starting with the individual's first request to
+      make a contribution to a group <a href="#deliverables">Deliverable</a>.
+      The section on <a href="#contrib">Contribution Mechanics</a> describes
+      how W3C expects to monitor these contribution requests.
+    </p>
+
+    <p>
+      The <a href="https://www.w3.org/Consortium/cepc/">W3C Code of
+      Ethics and Professional Conduct</a> applies to participation in
+      this group.
+    </p>
+
+    <h2 id="worklimit">
+      Work Limited to Charter Scope
+    </h2>
+    <p>
+      The group will not publish Specifications on topics other than those
```

Comment:
This means that every time we add a new specification we need to update the charter. But that is okay.